examples/CMakeLists.txt

```diff
@@ -72,6 +72,19 @@ if(GLOW_WITH_CPU)
                           ExecutionEngine
                           Graph
                           Importer)
+
+  add_executable(resnet-concurrent
+                   resnet-concurrent.cpp)
+  target_link_libraries(resnet-concurrent
+                        PRIVATE
+                          Backends
+                          CPUDeviceManager
+                          DeviceManager
+                          ExecutionEngine
+                          Graph
+                          Importer
+                          Optimizer)
+  target_include_directories(resnet-concurrent PUBLIC ${CMAKE_SOURCE_DIR}/lib)
```

Comment:
Can `CPUDeviceManager.h` be placed into glow/include instead of this? I assume other pieces would need that include as well later on.

Reply:
It would be weird if we did that for DeviceManagers but not for Backends though.

I think really we wouldn't expect to need to create a specific kind of device, instead we'd use the HostManager interface.

Don't really mind moving the header to include/ though I guess.

Reply:
Yeah, I guess either way it's a bit weird. Not feeling strongly either way.